JobConfig/common/su2020.fcl

```diff
@@ -0,0 +1,60 @@
+# -*- mode:tcl -*-
```

Comment:
i suggest to move this and all the su2020-specific configuration to its own directory (JobConfig/su2020).  Much of what is in JobConfig/common should also be moved to JobConfig/CDR, as it is specific to that and not really 'common', that's a project on my todo list.

Reply:
I've removed this file.